pynamodb/expressions/condition.py

```diff
@@ -0,0 +1,85 @@
+from pynamodb.expressions.util import get_value_placeholder, substitute_names
+
+
+class Path(object):
+
+    def __init__(self, path):
+        self.path = path
+
+    def __eq__(self, other):
+        return self._compare('=', other)
+
+    def __lt__(self, other):
+        return self._compare('<', other)
+
+    def __le__(self, other):
+        return self._compare('<=', other)
+
+    def __gt__(self, other):
+        return self._compare('>', other)
+
+    def __ge__(self, other):
+        return self._compare('>=', other)
+
+    def _compare(self, operator, value):
+        return Condition(self.path, operator, value)
+
+    def between(self, value1, value2):
+        # This seemed preferable to other options such as merging value1 <= attribute & attribute <= value2
+        # into one condition expression. DynamoDB only allows a single sort key comparison and having this
+        # work but similar expressions like value1 <= attribute & attribute < value2 fail seems too brittle.
+        return Between(self.path, value1, value2)
+
+    def startswith(self, prefix):
+        # A 'pythonic' replacement for begins_with to match string behavior (e.g. "foo".startswith("f"))
+        return BeginsWith(self.path, prefix)
+
+
+class Condition(object):
+    format_string = '{path} {operator} {0}'
```

Comment:
Yes, line 51 having a variable length arg array is mean to accommodate the `between` expression which takes two args. The `Between` class overwrites the format string as seen on Line 75. All the other expressions (thus far) only compare against a single value.